u2i-ci_utils.gemspec

```diff
@@ -36,11 +36,11 @@ Gem::Specification.new do |spec|
   if RUBY_PLATFORM == 'java'
     spec.platform = 'java'
   else
-    spec.add_dependency 'pronto', '~> 0.9'
-    spec.add_dependency 'pronto-rubocop', '~> 0.9'
-    spec.add_dependency 'pronto-brakeman', '~> 0.9'
-    spec.add_dependency 'pronto-flay', '~> 0.9'
-    spec.add_dependency 'pronto-reek', '~> 0.9'
-    spec.add_dependency 'pronto-fasterer', '~> 0.9'
+    spec.add_dependency 'pronto', '~> 0.9.5'
+    spec.add_dependency 'pronto-rubocop', '~> 0.9.0'
+    spec.add_dependency 'pronto-brakeman', '~> 0.9.1'
+    spec.add_dependency 'pronto-flay', '~> 0.9.0'
+    spec.add_dependency 'pronto-reek', '~> 0.9.0'
+    spec.add_dependency 'pronto-fasterer', '~> 0.9.0'
```

Comment:
Do this gem really require those versions or just they are just the newest? IMO gems should constraint versions of gems they can work with rather than the newest. And which particular version is used should be enforced by a project that used the particular gem. Otherwise it'd be a nightmare to find working versions of gems that may have common dependencies.

Reply:
without the bugfix version this gem simply accepts every single major version (they don't use the semantic versioning the right way), assuming they're going to stick to this versioning we could remove the versions completely and it'd give the same result

Reply:
in general you have right, and yes, let's not pass version 1.0.0 by default - I'll revert that change